src/ol/controls/RoutingControl.js

```diff
@@ -275,6 +248,33 @@ class RoutingControl extends Control {
     return null;
   }
 
+  drawViaPoint(viaPoint, idx) {
+    const pointFeature = new Feature();
+    pointFeature.set('viaPointIdx', idx);
+    if (Array.isArray(viaPoint)) {
+      pointFeature.setGeometry(new Point(viaPoint));
+      return this.routingLayer.olLayer.getSource().addFeature(pointFeature);
+    }
+    return fetch(
+      `https://api.geops.io/stops/v1/lookup/${viaPoint}?key=${this.stopsKey}`,
```

Comment:
Shouldn't the stops API URL also be configurable?

Reply:
Guess that makes sense